+dj/setup.m

```diff
@@ -0,0 +1,80 @@
+function setup(varargin)
```

Comment:
I like 'setup' better, but not sure if renaming to 'setup'  from 'setupDJ' is the best idea since matlab has a flat namespace.

Reply:
@ixcat Actually `setupDJ.m` was removed all together and this is a new function (They are located in different places). It was an undocumented method that was used internally only. Our tutorials in fact do not reference it and suggest to install via FileExchange (https://tutorials.datajoint.io/setting-up/datajoint-matlab.html). Reason for removing it as it did not follow proper encapsulating/scoping reference i.e. located outside of the `+dj` package.

Reply:
ah ok - makes sense in that case :) thanks for clarification